content/client/Client Deployment/_index.en.md

```diff
@@ -145,8 +145,9 @@ open -n /Applications/RustDesk.app
 ```sh
 #!/bin/bash
 
-# Assign the value "XYZ" to the password variable
-rustdesk_password="XYZ"
+# Assign a random value to the password variable
+rustdesk_password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 8 | head -n 1)
```

Comment:
for what?

Reply:
why do not leave permanent password empty?

Reply:
Follow some configuration template file used by postgres or sshd etc.

Reply:
It is ok if the user can get the random password somewhere.

Reply:
they dont set passwords, this is an install script not a config file. its moreso that I have something to point people to with support. I was going to use the same template for windows, linux and mac but havent got there yet and better to have something

Reply:
> It is ok if the user can get the random password somewhere.

its output at the end of the script, rmms, intune etc have collectors so would pick that up

Reply:
Understand now

Reply:
have updated the description a bit to make it more obvious what its installed for, once I am home again I will do some video walk throughs

Reply:
Great